src/index.js

```diff
@@ -0,0 +1,79 @@
+import './style.css';
+
+class TaskList {
+  constructor() {
+    this.size = 0;
+    this.ul = document.getElementById('task-list');
+    this.savedList = JSON.parse(localStorage.getItem('savedList')) || [];
+    // this.displayAllTask();
+  }
+
+  clearList() {
+    this.savedList = [];
+    localStorage.setItem('savedList', JSON.stringify(this.savedList));
+    this.displayAllTask();
+  }
+
+  addIndex() {
+    if (this.savedList.length < 1) {
+      return this.index;
+    }
+    return this.savedList[this.savedList.length - 1].index + 1;
+  }
+
+  addTask(book) {
+    this.savedList.push(book);
+    localStorage.setItem('savedList', JSON.stringify(this.savedList));
+    this.size += 1;
+  }
+
+  displayAllTask() {
+    this.ul.innerHTML = '';
+
+    this.savedList.forEach((task) => {
+      const li = document.createElement('li');
+      li.className = 'list-item';
+
+      const checkbox = document.createElement('input');
+      checkbox.type = 'checkbox';
+      checkbox.className = 'checkbox';
+      const p = document.createElement('p');
+      const index = document.createElement('spand');
+      index.className = 'index';
+      const button = document.createElement('i');
+      button.classList.add('fa', 'fa-ellipsis-v', 'icon');
+
+      p.innerHTML = task.description;
+      index.innerHTML = task.index;
```

Comment:
- [ ] You are supposed to have an array of some simple to do tasks (array of objects) 
with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]
- [ ] As the previous reviewer mentioned, on page load, the task array is rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task, currently when I load the page it doesn't show any task please work on that

Reply:
Thank you